### DISCUSSION TITLE
請教 Steam OS 的安裝方法

### DISCUSSION THREAD

**Discussion by kai004** [GpointChen/FFXIVChnTextPatch-GP#246] [General]
我對Arch Linux不熟 也沒用過Gradle 所以光看Linux Builld的安裝說明照做實在有點困難

1. univocity-parsers-2.9.1.jar是否該放入到gradle的lib資料夾裡面?

2. 配置環境變量的步驟  `export PATH="/path/to/gradle-6.8.3/bin:$PATH"` 是否將其path改成gradle解壓後的資料夾?
例如`export PATH="/home/deck/Gradle/gradle-6.8.3/bin:$PATH"`

3. 以上指令是否在konsole上面執行?

4.  `在 Terminal 中，執行 gradle run`   <<<最後的步驟實在看不懂 
是否要在FFXIVChnTextPatch解壓後的資料夾裡使用konsole 來執行 `gradle run FFXIVChnTextPatchGP.exe` ?
但是執行以上指令的話會出現 `Failure: Build failed with an exception. Task "run" not found in root project FFXIVChnTextPatch-20221102`

**Comment by chun1324** [comment]
我不清楚Linux/SteamOS執行漢化器的方法
不過如果你只是想要使用漢化檔, 而且你有Windows, 可以先在Windows上面完成漢化, 然後使用SteamDeck的桌面模式, 複製下面這些檔案到 /home/deck/.steam/steam/steamapps/common/FINAL FANTASY XIV Online/game/sqpack/ffxiv
其中 .steam為隱藏項目, 按下 Ctrl + H 顯示
(另外: 如果使用第三方啟動器XIVLauncher: ~/.xlcore/ffxiv/game/sqpack/ffxiv)
000000.win32.dat0
000000.win32.index
000000.win32.index2
0a0000.win32.dat0
0a0000.win32.index
0a0000.win32.index2

**Reply by kai004** [reply]
謝謝你的回覆. 但我還是不太懂你的意思. 目前我已經在windows上執行了漢化檔 並把其內容傳到steam deck上. 但我不知道該如何把這些檔案放置到正確的資料夾裡. 我有使用XIVLauncher, 其資料夾裡也有你列出來的檔案. 我的 ../steamapps/common/FINAL FANTASY XIV Online/game/ 裡是空的 沒有sqpack/ffxiv

**Reply by chun1324** [reply]
如果你使用的是XIVLauncher, 則無必要複製到steamapps裡面, 應該放到 /home/deck/.xlcore/ffxiv/game/sqpack/ffxiv
.xlcore 是隱藏項目, 按下 Ctrl + H 顯示

**Reply by kai004** [reply]
我之前沒有理解那6個檔案是漢化器生成的遊戲文檔所以很疑惑 但我現在成功漢化了 謝謝.
